.github/workflows/esphome_build.yaml

```
@@ -23,19 +23,21 @@ jobs:
         uses: esphome/build-action@v6
         with:
-          yaml-file: config/nexxtender.yaml
+          yaml-file: nexxtender.yaml
+        working-directory: config
 
       - name: Build ESPHOME esp32-s3-devkitc-1
         uses: esphome/build-action@v6
         with:
-          yaml-file: config/nexxtender.s3.yaml
+          yaml-file: nexxtender.s3.yaml
+        working-directory: config
```

actionlint (1.7.4)

"working-directory" is not available with "uses". it is only available with "run"

(syntax-check)